deploy/logrotate/dht-tcm-backup

```diff
@@ -0,0 +1,10 @@
+/srv/dhtnode/dhtnode-?/data/*.tcm {
+  rotate 7
+  daily
+  compress
+  copy
+  dateext
+  notifempty
```

Comment:
Not sure if it matters that we're using `notifempty` here. I don't think so.

Reply:
I don't think it matters as well.